Include/pyportosf.h

```diff
@@ -0,0 +1,72 @@
+#if defined(__osf__) && defined(__arch64__)
+
+// Native cc headers require requesting pieces of C99.
+// Native cc support is hypothetical. gcc4 is being used.
+#define __STDC_CONSTANT_MACROS 1
+#define __STDC_LIMIT_MACROS 1
+#define __STDC_FORMAT_MACROS 1
```

Comment:
These are ok for all systems. Some others even require at least one of them.

Reply:
Kinda like this: https://github.com/mono/mono/blob/e44d84d10a0a6bfe8376cec30582c5c93fd3a92f/mono/eglib/glib.h#L9